src/ert/analysis/_es_update.py

```diff
@@ -349,6 +351,11 @@ def _load_observations_and_responses(
     ens_mean_mask = abs(observations - ens_mean) <= alpha * (ens_std + errors)
     obs_mask = np.logical_and(ens_mean_mask, ens_std_mask)
 
+    if misfit_process:
+        errors[obs_mask] *= misfit_preprocessor.main(S[obs_mask], errors[obs_mask])
+        ens_mean_mask = abs(observations - ens_mean) <= alpha * (ens_std + errors)
+        obs_mask = np.logical_and(ens_mean_mask, ens_std_mask)
```

Comment:
Should we move this outside the loading function to make this operation more explicit?

Reply:
The full processing? We could, but then we should move creation of the snapshot as well, because that is the only trace that this has taken place. Did it like this to change as little as possible, but agree it is cleaner higher in the stack. Not sure what the least evil is.

Reply:
> Could we report the original snapshot and the updated?

How about we always report the scaling factors in all snapshots? So for esmd you will for example get:
```
+- 1.0 (5.0 * 2)
```
where 2 is an example scaling factor from esmda? And if using misfit preprocessor we still just provide 1 value for scaling, but it will be different for different observations.

Reply:
Good idea!